pages/_app.js

```diff
@@ -1,5 +1,11 @@
+import { grommet, Grommet } from 'grommet';
+
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <Grommet theme={grommet}>
+      <Component {...pageProps} />
+    </Grommet>
+  );
```

Comment:
I think we can leave <Grommet> wrapper in the index.js file

Reply:
@jcfilben  Leave that in the `/pages/index.js` file is not wrong, but users will have to do it for every new page since `/pages/index.js` it's only the `/` page.

I.e. if I want to create an "about" page for my project, I'll have to remember that I have to wrap my `<AboutPage />` component into the `<Grommet />` again.

We can use the current Grommet site project as a reference. The [`<Grommet />` context is on the `App.js` file](https://github.com/grommet/grommet-site/blob/master/src/App.js#L49), so on the [about page, we don't have to do it again](https://github.com/grommet/grommet-site/blob/master/src/screens/About.js).

What do you think? Should I change it?

Reply:
Good explanation, I agree